fov_processing_pipeline/data.py

```diff
@@ -10,6 +8,8 @@
 
 
 def get_cell_data():
+    import lkaccess
```

Comment:
I would just add a comment or TODO here to the effect of `Move this to top-level imports if/when lkaccess becomes open source`